src/contents/styles.css

```diff
@@ -1,120 +1,48 @@
 /* TODO: refactor */
 
-body {
-  &:not([data-aws_masking_disabled="true"]) {
-    /* account id */
-    &[data-aws_masking_account_ids="true"] {
-      span[data-aws_masking_account_id="true"] {
-        filter: blur(5px);
-      }
-
-      &.awsui-polaris-dark-mode[data-aws_masking_inputs="true"] {
-        textarea,
-        input[type="text"],
-        input[type="search"] {
-          &[data-aws_masking_account_id="true"] {
-            color: transparent !important;
-            text-shadow: 0px 0px 8px rgba(255, 255, 255, 0.8) !important;
-          }
-        }
-      }
-
-      &[data-aws_masking_inputs="true"] {
-        textarea,
-        input[type="text"],
-        input[type="search"] {
-          &[data-aws_masking_account_id="true"] {
-            color: transparent !important;
-            text-shadow: 0px 0px 8px rgba(0, 0, 0, 0.5) !important;
-          }
-        }
-      }
-    }
-
-    /* arn */
-    &[data-aws_masking_arns="true"] {
-      span[data-aws_masking_arn="true"] {
-        filter: blur(5px);
-      }
-
-      &.awsui-polaris-dark-mode[data-aws_masking_inputs="true"] {
-        textarea,
-        input[type="text"],
-        input[type="search"] {
-          &[data-aws_masking_arn="true"] {
-            color: transparent !important;
-            text-shadow: 0px 0px 8px rgba(255, 255, 255, 0.8) !important;
-          }
-        }
-      }
-
-      &[data-aws_masking_inputs="true"] {
-        textarea,
-        input[type="text"],
-        input[type="search"] {
-          &[data-aws_masking_arn="true"] {
-            color: transparent !important;
-            text-shadow: 0px 0px 8px rgba(0, 0, 0, 0.5) !important;
-          }
-        }
-      }
-    }
-
-    /* access key id */
-    &[data-aws_masking_access_key_ids="true"] {
-      span[data-aws_masking_access_key_id="true"] {
-        filter: blur(5px);
-      }
-
-      &.awsui-polaris-dark-mode[data-aws_masking_inputs="true"] {
-        textarea,
-        input[type="text"],
-        input[type="search"] {
-          &[data-aws_masking_access_key_id="true"] {
-            color: transparent !important;
-            text-shadow: 0px 0px 8px rgba(255, 255, 255, 0.8) !important;
-          }
-        }
-      }
+body:not([data-aws_masking_disabled="true"]) {
```

Comment:
Since the check for the `data-aws_masking_*s` value in styles.css has been removed, it is no longer possible to switch masks for each item.
However, ultimately, trying to add that check back in would likely make the code messy. I'm wondering if there's a cleaner way to implement this functionality... 🤔

Reply:
Shoot, you're right. I actually *had* a solution that addresses the problem you're talking about and lost the commit. I just noticed the switches are broken.

When I get some time I'll try to fix this again.